### DISCUSSION TITLE
Did v4.4.5 break create/combine with TypeScript?

### DISCUSSION THREAD

**Discussion by rodrigocfd** [pmndrs/zustand#2172] [Bug report] [answered]
I'm using this example from the [TypeScript guide](https://github.com/pmndrs/zustand/blob/main/docs/guides/typescript.md):

```ts
import { create } from 'zustand'
import { combine } from 'zustand/middleware'

const useBearStore = create(
  combine({ bears: 0 }, (set) => ({
    increase: (by: number) => set((state) => ({ bears: state.bears + by })),
  }))
)
```

It worked until today, when I updated to v4.4.5... is this a bug?

**Comment by dbritto-dev** [comment, marked as answer]
yes, here's the fix -> https://github.com/pmndrs/zustand/pull/2170. BTW, would you mind trying the local instructions (https://ci.codesandbox.io/status/pmndrs/zustand/pr/2170/builds/434064)?

**Comment by dai-shi** [comment]
See also #2163
